packages/migrate/src/commands/DbPull.ts

```diff
@@ -391,19 +398,22 @@ Or run this command with the ${green(
     const introspectionWarningsMessage = this.getWarningMessage(introspectionWarnings)
 
     if (args['--print']) {
-      process.stdout.write(introspectionSchema + '\n')
+      printIntrospectedSchema(introspectionSchema, process.stdout)
 
       if (introspectionWarningsMessage.trim().length > 0) {
         // Replace make it a // comment block
         console.error(introspectionWarningsMessage.replace(/(\n)/gm, '\n// '))
       }
     } else {
       schemaPath = schemaPath || 'schema.prisma'
-      fs.writeFileSync(schemaPath, introspectionSchema)
+      if (args['--force']) {
+        await removeSchemaFiles(schema)
+      }
+      await saveSchemaFiles(introspectionSchema)
+
+      const { modelsCount, typesCount } = countModelsAndTypes(introspectionSchema)
 
-      const modelsCount = (introspectionSchema.match(/^model\s+/gm) || []).length
       const modelsCountMessage = `${modelsCount} ${modelsCount > 1 ? 'models' : 'model'}`
-      const typesCount = (introspectionSchema.match(/^type\s+/gm) || []).length
       const typesCountMessage = `${typesCount} ${typesCount > 1 ? 'embedded documents' : 'embedded document'}`
       let modelsAndTypesMessage: string
       if (typesCount > 0) {
```

Comment:
(could not add a comment on the right line)

Below at line 431
```
      introspectionSpinner.success(`Introspected ${modelsAndTypesCountMessage} into ${underline(
        path.relative(process.cwd(), schemaPath),
```

This is no correct anymore when it's a multi-file schema, because now it just says that everything is saved in one schema file.

Reply:
`schemaPath` is a path to directory in this case, so message just says the schema was saved into a directory, which is correct.